sdk/src/main/java/com/karhoo/sdk/analytics/Event.kt

```diff
@@ -43,7 +43,25 @@ enum class Event constructor(val value: String) {
     USER_PROFILE_SAVE_PRESSED("user_profile_save_pressed"),
     USER_PROFILE_UPDATE_SUCCESS("user_profile_update_success"),
     USER_PROFILE_UPDATE_FAILED("user_profile_update_failed"),
+
+    //PAYMENT
+    CARD_AUTHORISATION_SUCCESS("CARD_AUTHORISATION_SUCCESS"),
+    CARD_AUTHORISATION_FAILURE("CARD_AUTHORISATION_FAILURE"),
     PAYMENT_FAILED("payment_failed"),
+
+    //LOYALTY
+    LOYALTY_STATUS_REQUESTED("LOYALTY_STATUS_REQUESTED"),
+    LOYALTY_PREAUTH_FAILED("LOYALTY_PREAUTH_FAILURE"),
```

Comment:
why do we have failed and failure for the same event? I see that for card auth it's failure all the way through, for example. please change it to failure here as well

Reply:
Corrected